README.md

```diff
@@ -517,6 +517,15 @@ not supported to write them to individual databases. The first point written
 defines the database, other values are silently ignored!
 
 
+## Contributing
+
+##### Custom influxdb test connection
+
+```
+export INSTREAM_HOST=localhost
+export INSTREAM_HTTP_PORT=8086
```

Comment:
Might it be a good alternative to not encourage exporting variables to the shell environment and instead use single invocation definition?

```shell
INSTREAM_HOST=localhost INSTREAM_HTTP_PORT=8086 mix test
```

However I think this may be completely irrelevant. Someone using a separate server (container, virtual machine, ...) to run the test against should be familiar with both ways.